scripts/run_codegen.py

```diff
@@ -40,7 +40,7 @@ def main() -> None:
     # Run codegen.
     try:
         subprocess.run(
-            ["ariadne-codegen"],
+            ["ariadne-codegen", "client"],
```

Comment:
I imagine this would fix the codegen problem I was having, I'll try again running the script but judging for the traceback this looks related.

Reply:
It's weird that it just started happening. It wasn't occurring for me until now.